definitions/types/azure-resource-group.json

```
@@ -5,5 +5,9 @@
   "description": "Azure Resource Group",
   "examples": [
     "resource-group-name"
-  ]
+  ],
+  "pattern": "^[a-zA-Z0-9-_.()]+[a-zA-Z0-9-_()]+$",
```

```suggestion
  "pattern": "^[a-zA-Z0-9-_.()]+[a-zA-Z0-9-_()]$",
```
Syntactically this is the same, but it's clearer to the reader by removing the last `+`